sarazaToussaint/Way_Farer
# #167222388 ft(create trip):Styling admin ui, create trip form [Start 167222388]

#### What does this PR do?
    Has a create trip form on admin page
#### Description of Task to be completed?
Has the following input fild to complete
seating capacity
bus licence number
origin 
destination
date trip
fare
#### How should this be manually tested?
After minimizing your screen device
you should see that the create trip form is responsive

#### Any background context you want to provide?
     There is a goal of making ui with no front end framework 
### What are the relevant pivotal tracker stories?
 #167222388
